package_python_function/packager.py

```diff
@@ -45,10 +48,18 @@ def zip_all_dependencies(self, target_path: Path) -> None:
             def zip_dir(path: Path) -> None:
                 for item in path.iterdir():
                     if item.is_dir():
-                        zip_dir(item)
+                        if item.name not in self.DIRS_TO_EXCLUDE:
+                            zip_dir(item)
                     else:
-                        self._uncompressed_bytes += item.stat().st_size
-                        zip_file.write_reproducibly(item, item.relative_to(self.input_path))
+                        is_excluded_by_extension = item.suffix in self.EXTENSIONS_TO_EXCLUDE
+                        is_excluded_dist_info_file = (
+                            item.name in self.DIST_INFO_FILES_TO_EXCLUDE
+                            if ".dist-info" in item.parent.name
```

Comment:
nit: Probably best to use `item.parent.name.endswith(".dist-info")`

Reply:
ooo nice call